exporter.go

```diff
@@ -305,6 +306,7 @@ func NewRedisExporter(redisURI string, opts ExporterOptions) (*Exporter, error)
 		"slowlog_length":                       {txt: `Total slowlog`},
 		"start_time_seconds":                   {txt: "Start time of the Redis instance since unix epoch in seconds."},
 		"up":                                   {txt: "Information about the Redis instance"},
+		"connected_clients_details":            {txt: "Details about connected clients", lbls: []string{"host", "port", "name", "fd", "age", "idle", "flags", "db", "sub", "psub", "multi", "qbuf", "qbuffree", "obl", "oll", "omem", "events", "cmd"}},
```

Comment:
Let's reduce the number of labels a bit, not all of them are super useful and having that many (esp fsasdt changing ones like age, idle, etc) will blow up the namespace a lot.

Let's go with `"host", "port", "name", "flags", "db"` for now.

Reply:
I like it

Reply:
age is probably good and idle for showing old clients and clients that are doing nothing